app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ReviewsController < ApplicationController
+  before_action :require_user
+
+  def create
+    @review = Review.new(params_review)
+    @review.user_id = current_user.id
+    @review.video_id = params[:video_id]
+    @review.rating = params[:Rating]
+
+
+    if @review.save
+      flash[:notice] = "Your review has been successfully saved."
+      redirect_to video_path(params[:video_id])
+    else
+      @video = Video.find(params[:video_id])
+      @reviews = Review.all.select{|review| review.video_id == @video.id}
```

Comment:
This could be rewritten to `@reviews = @video.reviews.reload` to build all reviews associated with `@video`. `Review.all` is going to be inefficient when you get a lot of reviews in database.